### ISSUE TITLE
НЕкорректно отображается ПДФ-файл

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#144]
```
What steps will reproduce the problem?
1. НЕкорректно отображается ПДФ-файл - сжат 
повертикали.
2.Устройство HTC Incredible S
Андроид 2.3.5. НТС сенс 3.0.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?


What device are you using?


Please provide any additional information below.

```

Original issue reported on code.google.com by `7059...@gmail.com` on 3 Feb 2012 at 11:26

**Comment by GoogleCodeExporter** [comment]
```
Не понял как прикрепить сюда файл. Вот 
ссылка.

http://zalil.ru/32635957
```

Original comment by `7059...@gmail.com` on 3 Feb 2012 at 11:29

**Comment by GoogleCodeExporter** [comment]
```
Fixed with custom DPI in 1.4.0
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 4 Apr 2012 at 10:48
- Changed state: **Fixed**
